Jellyfin.Server/Program.cs

```diff
@@ -76,6 +76,9 @@ private static async Task StartApp(StartupOptions options)
             // $JELLYFIN_LOG_DIR needs to be set for the logger configuration manager
             Environment.SetEnvironmentVariable("JELLYFIN_LOG_DIR", appPaths.LogDirectoryPath);
 
+            // As far as Jellyfin is concerned the Environment path should also include wherever Jellyfin is running from
+            Environment.SetEnvironmentVariable("PATH", Environment.GetEnvironmentVariable("PATH") + Path.PathSeparator + Environment.CurrentDirectory);
```

Comment:
this doesn't feel right TBH... maybe it's better to add path to jellyfin executable, rather than current working directory?..

Reply:
This has been changed